selfdrive/car/docs_definitions.py

```diff
@@ -146,6 +146,14 @@ class Harness(Enum):
   hyundai_o = "Hyundai O"
   custom = "Developer"
   obd_ii = "OBD-II"
+  gm_cam = "GM Camera" 
```

Comment:
Wasn't sure what to use for harness - developer isn't accurate and would have people ordering wires they don't need - this is why I didn't want to use custom

Reply:
perhaps we just call this "gm"
```suggestion
  gm = "GM" 
```